moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -116,7 +105,11 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       defaultOperator: "string_equals_case_insensitive",
       lookup: {
         table_name: "moped_types",
-        field_name: "type_name",
+        getOptionLabel: (option) => option.type_name,
```

Comment:
I added this function to the column-specific configuration so we can customize what keys the label uses out of the option objects for the dropdown choices. We used to set a `field_name` here in the config which was used but it only supported a single key. We need more than one for columns like **Team** which is first name + last name.

Reply:
nice!